runtime/cudaq/platform/default/rest/helpers/braket/braket.yml

```diff
@@ -0,0 +1,30 @@
+
```

Comment:
Missing license header

(Fix for CI failure - https://github.com/NVIDIA/cuda-quantum/actions/runs/11630638214/job/32395476520?pr=2337)

```
# ============================================================================ #
# Copyright (c) 2022 - 2024 NVIDIA Corporation & Affiliates.                   #
# All rights reserved.                                                         #
#                                                                              #
# This source code and the accompanying materials are made available under     #
# the terms of the Apache License 2.0 which accompanies this distribution.     #
# ============================================================================ #
```